packages/di/test/di.test.tsx

```diff
@@ -108,11 +112,11 @@ describe('@bem-react/di', () => {
     describe('useRegistry', () => {
       test('should pull component from registry', () => {
         const registry = new Registry({ id: 'registry' })
-        const Element: React.FC = ({ children }) => <span>{children}</span>
+        const Element: React.FC<BaseProps> = ({ children }) => <span>{children}</span>
```

Comment:
Now `children` does not exist in type `FC` by default

https://stackoverflow.com/questions/71788254/react-18-typescript-children-fc/71800185#71800185